utils_general/aa-countries.R

```diff
@@ -0,0 +1,75 @@
+###
+# Install packages
+###
+
+library(tidyverse)
+#library(rvest)
+#library(magrittr)
+#library(ggmap)
+#library(stringr)
+
+###
+# get world map
+###
+
+world_map <- map_data("world")
+
+###
+# identify AA pilots
+###
+
+# list countries and shocks
+drought <- c("Somalia", "Ethiopia", "Niger", "Burkina Faso")
+#dry_spells <- c("Malawi")
+#floods <- c("Bangladesh", "Nepal", "Malawi", "Chad", "South Sudan")
```

Comment:
remove commented lines

Reply:
I was leaving them in so we can easily update it / change how we represent MWI. I guess if the overall structure & pattern are clear I can remove them.

Reply:
to me the overall structure is clear